apidoc/Annotation.yml

```diff
@@ -76,6 +76,131 @@ properties:
     type: Titanium.UI.View
     since: "3.1.0"
 
+  - name: showMarker
+    summary: |
+        Boolean to show an annotation view that displays a balloon-shaped 
+        marker at the designated location.
+    description: |
+        Must be set during creation. This is ignored if the customView property is set.
+    type: Boolean
+    default: false
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "11.0"} }
+
+  - name: markerGlyphText
+    summary: The text to display in the marker balloon.
+    description: |
+        Use this property or the markerImage property to specify the marker balloon content. 
```

Comment:
`markerImage ` => `<Modules.Map.Annotation.markerImage>`, so it links to the actual property automatically. Please also check other cases where this would make sense.

Reply:
Done